ontology/yaml/resources/METERS/entity_types/EM.yaml

```diff
@@ -97,3 +97,11 @@ EM_US_SVL_TC1_1:
   uses:
   - average_line_current_sensor
   - average_inter_line_voltage_sensor
+#AU-SYD-ODI meter types
+EM_SE_PM5100: 
```

Comment:
please change name to EM_NONCANONICAL_1 or similar (remove manufacturer name).

Reply:
Changed to EM_NONCANONICAL_1